pulsar-jms/src/test/resources/README.md

```diff
@@ -0,0 +1,24 @@
+# TLS Certificate Testing
+
+In order to run complete and correct tests for TLS, there are generated certificates in this directory. They were
+generated using the following script:
+
+```shell
+set -e
+
+openssl genrsa -out ca.key 2048
+
+openssl req -x509 -subj '/CN=localhost' -new -nodes -key ca.key \
```

Comment:
It's not really an issue in this PR, since this is a problem in most Pulsar TLS instructions that SAN isn't used for certificates. I've always wondered about the mix of using openssl and keytool. keytool alone could create proper certificates. With openssl, adding a SAN is possible, but it's a bit awkward. Here's an example from a script for renewing Pulsar test certs:
https://github.com/apache/pulsar/blob/6cc30c93a05bb16b7ef62ffebc15ceb5e6efb502/build/regenerate_certs_for_tests.sh#L37

My assumption is that using keytool alone would be a more portable way.
By googling, I found this example: https://ultimatesecurity.pro/post/san-certificate/ 
On Stackoverflow, https://stackoverflow.com/questions/8744607/how-to-add-subject-alernative-name-to-ssl-certs